@here/harp-debug-datasource/tsconfig.json

```diff
@@ -2,13 +2,30 @@
     "compilerOptions": {
         "target": "es2017",
         "module": "commonjs",
+        "outDir": "dist/cjs",
         "sourceMap": true,
         "declaration": true,
+        "declarationMap": true,
+        "composite": true,
+        "strict": true,
         "strictNullChecks": true,
         "noImplicitAny": true,
         "noImplicitReturns": true,
         "noImplicitThis": true,
-        "noUnusedLocals": false,
-        "downlevelIteration": true
-    }
+        "noUnusedLocals": false
+    },
+    "references": [
```

Comment:
You mentioned you have script to generate these references. It would be great to have it in this patch.